openmsupply/open-msupply
# #1526 Show patient name in encounter breadcrumb

Fixes #1526

# 👩🏻‍💻 What does this PR do? 
 

Solution is essentially [Option 2 from the issue](https://github.com/openmsupply/open-msupply/issues/1526#issuecomment-1507874264) -- just a small amendment to the "Breadcrumb" module to accept a "suffix" that can be a pre-build React element as well as (as it is now) a simple string. This allows us to set a multi-part, hyperlinked suffix for this particular case.

# 🧪 How has/should this change been tested? 


- Load an encounter. The Patient name should be present in the Breadcrumb and should be linked to the patient page when clicked:
![Screenshot 2023-04-19 at 11 26 43 AM](https://user-images.githubusercontent.com/5456533/232926235-aa5d5600-44d7-4b62-9b73-b9235a32040f.png)

- Make sure existing behaviour for Breadcrumbs is unchanged

## 💌 Any notes for the reviewer?

We should probably make the changes to "useBreadcrumb" and "Breadcrumbs" on `develop` as well? I'll make a separate PR for that with just those changes once this is approved.
